code/__defines/access.dm

```diff
@@ -688,7 +688,7 @@
 #define ACCESS_IDRIS 220
 /datum/access/idris
 	id = ACCESS_IDRIS
-	desc = "Idris Vault Ship"
+	desc = "Idris Ship Crew Member"
```

Comment:
If you're changing the desc on this, please make sure to change it on the corpses on idris_wreck.dm, as corpse IDs use desc to set access.

Reply:
changed